src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -106,6 +108,7 @@ describe('manage nsfs cli bucket flow', () => {
             await fs_utils.create_fresh_path(`${config_root}/${CONFIG_SUBDIRS.BUCKETS}`);
             await fs_utils.create_fresh_path(root_path);
             await fs_utils.create_fresh_path(bucket_storage_path);
+            await fs_utils.create_fresh_path(path_to_json_bucket_options_dir);
```

Comment:
"json bucket options" is related to "from_file" in bucket add and not related to delete, please change it.